lti_consumer/lti_1p3/ags.py

```diff
@@ -60,11 +63,17 @@ def get_lti_ags_launch_claim(self):
         """
         Returns LTI AGS Claim to be injected in the LTI launch message.
         """
+
+        claim_values = {
+            "scope": self.get_available_scopes(),
+            "lineitems": self.lineitems_url,
+        }
+
+        if self.lineitem_url:
+            claim_values["lineitem"] = self.lineitem_url
+
         ags_claim = {
```

Comment:
@shimulch Minor nitpickinig: just return the dict instead of assigning it to a variable.
```suggestion
        return {
```

Reply:
Fixed